### DISCUSSION TITLE
Configure LDAP client on Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by jdiekhoff** [lldap/lldap#864] [Q&A] [answered]
I'm not sure if what I'm doing is supported, but here's my scenario:

I have setup lldap as a docker container and have configured an LDAP client on Ubuntu 22.04 such that I can run `ldapsearch` queries against the lldap server, I see the users & groups I've setup in the UI. However, despite the dozens of tutorials I've tried to follow on setting up Ubuntu ldap clients, I can never login or switch to one of the lldap users. 

From what I've read of the client configuration, lldap seems to be more targeted for use for applications, not necessarily for users directly on a server. Is my scenario of configuring an ldap client to use lldap for linux users possible, or am I barking up the wrong tree?

**Comment by martadinata666** [comment, marked as answer]
system user login, likely PAM suport like this one? https://github.com/lldap/lldap/issues/294

**Comment by jdiekhoff** [comment]
Yes, PAM support is exactly what I'm looking for. I will take a look at that issue.

**Reply by jakob42** [reply]
You'll need to wait for custom attributes (or test now with a dev build). PAM will need some uids/gids, shell and so on.

**Reply by nitnelave** [reply]
custom attributes are not in stable, but they're in latest (or daily builds). You can use https://github.com/Zepmann/lldap-cli to manage them, while we get the frontend sorted.
